src/main/kotlin/org/rust/openapiext/CommandLineExt.kt

```diff
@@ -36,32 +38,29 @@ fun GeneralCommandLine(path: Path, withSudo: Boolean = false, vararg args: Strin
 fun GeneralCommandLine.withWorkDirectory(path: Path?) = withWorkDirectory(path?.systemIndependentPath)
 
 fun GeneralCommandLine.execute(timeoutInMilliseconds: Int?): ProcessOutput? {
```

Comment:
Is there a reason why don't you use `RsProcessResult<ProcessOutput>` here?

Reply:
Well, I don't know. I just preserved the signature and semantics of this method.